packages/common/src/env.ts

```diff
@@ -14,7 +14,7 @@ export type Env = {
 
 const env: Env = {
   APP_VERSION: '',
-  APP_API_BASE_URL: 'https://cdn.jwplayer.com',
+  APP_API_BASE_URL: 'https://cdn-dev.jwplayer.com',
```

Comment:
```suggestion
  APP_API_BASE_URL: 'https://cdn.jwplayer.com',
```

Reply:
That way it'll fail fetching the config.